test/verify/check-kdump

```diff
@@ -195,6 +195,20 @@ class TestKdump(KdumpHelpers):
         self.login_and_go("/kdump")
         b.wait_visible("#app")
 
+        # Check defaults
+        current = m.execute("grep '^core_collector' /etc/kdump.conf").strip()
+        m.execute("sed -i /^core_collector/d /etc/kdump.conf")
+        # Drop the custom path so we can make sure our changes are synced to JavaScript
+        m.execute("sed -i 's#^path /var/crash#path /var/tmp#' /etc/kdump.conf")
+        b.wait_text("#kdump-change-target", "locally in /var/tmp")
+        b.click("#kdump-change-target")
+        b.wait_visible("#kdump-settings-dialog")
+        b.set_input_text("#kdump-settings-local-directory", "/var/crash")
+        b.click("button:contains('Save')")
+        b.wait_not_present("#kdump-settings-dialog")
+        new = m.execute("grep '^core_collector' /etc/kdump.conf").strip()
+        self.assertEqual(current, new)
```

Comment:
The commit message talks about validating the hardcoded default value. I don't see this here? What does it mean?

Reply:
The idea of the test is to get the default value from `/etc/kdump.conf` then drop this from the config as if the admin changed it and then save from Cockpit and verify that we set the default which is the kdump package default.

This way we would detect if the kexec-tools package changed their default configuration

Reply:
Ah, I understand now. Thanks!